### PR TITLE
Security stubs

### PR DESCRIPTION
**Pull request by milesflo**

### CODE REVIEW

File: security/securing-unix-the-osi-model/identify-the-physical-layer.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+---
+## Quiz
+
+headline: What role does this Layer play in the OSI 7 Layer Model?
+
+question:| Physical Layer
+
+answers:
+
+* Providing transmission between end devices via voltage or wavelengths.
```
Comment (lizTheDeveloper) [inline security/securing-unix-the-osi-model/identify-the-physical-layer.md]: for quiz questions, these have to be dashes

**Comment by lizTheDeveloper** [comment]
approved

**Comment by milesflo** [comment]
This PR's redundant. See https://github.com/enkidevs/curriculum/pull/480

**Comment by lizTheDeveloper** [comment]
This lives in PR #485 now (gotta make some changes to support creating a topic), so closing this one.

**Pull request closed without merging**